packages/build/jest/config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
   moduleNameMapper: {
     // mock all imports to asset files
     '\\.(css|scss|stylesheet)$': path.join(__dirname, 'mockStyles.js'),
-    '\\.(png|svg|yaml)$': path.join(__dirname, 'mockFiles.js'),
+    '\\.(png|svg|yaml|yaml\\?raw)$': path.join(__dirname, 'mockFiles.js'),
```

Comment:
I'm not familiar with the `yaml?raw` - what's going on here?

Reply:
You can use `resourceQuery` to match with `?whatever` on imports, and the webpack docs recommend changing `raw-loader` to use a `resourceQuery` match instead, and specify it as a `asset/source` type - https://webpack.js.org/guides/asset-modules/#replacing-inline-loader-syntax